build/packages.targets

```diff
@@ -23,6 +23,8 @@
         <PackageDownload Update="NuGet.Core" Version="[$(NuGetCoreV2Version)]" />
         <!-- PackageReference -->
         <PackageReference Update="Lucene.Net" Version="3.0.3" />
+        <PackageReference Update="Markdig.Signed" Version="0.22.0" />
```

Comment:
Do we vet if new package is ok to include?

Reply:
There could be security and license concerns.

Reply:
I wondered this as well. This particular package is also being used by GitHub VS components (though, older versions), so I assume @jebriede knows more and will let us know what he knows.